@here/harp-vectortile-datasource/lib/VectorTileDataEmitter.ts

```diff
@@ -226,6 +226,17 @@ enum LineType {
 type TexCoordsFunction = (tilePos: THREE.Vector2, tileExtents: number) => THREE.Vector2;
 const tmpColor = new THREE.Color();
 
+/**
+ * Options for VectorTileDataEmitter, @see {@link DecoderOptions} and {@link OmvDecoderOptions}.
```

Comment:
This doesn't seem to be rendered correctly, please try this locally and fix.

Reply:
DecoderOptions works. OmvDecoderOptions doesn't because it's marked as "@hidden". I've changed it to "@internal" instead, so that we can see it in our documentation but still can omit it from public documentation by not exporting internals.